hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/queries/file/HapiGetFileContents.java

```diff
@@ -181,7 +181,7 @@ protected void submitWith(HapiSpec spec, Transaction payment) throws Throwable {
                 configList
                         .getNameValueList()
                         .forEach(setting ->
-                                entries.add(String.format("\n  %s=%s", setting.getName(), setting.getValue())));
+                                entries.add(String.format("%n  %s=%s", setting.getName(), setting.getValue())));
```

Comment:
It seems like a \n newline character is needed so that the entries don't all run together when appended.

Reply:
correct that why %n is there
[https://stackoverflow.com/questions/1883345/whats-up-with-javas-n-in-printf](url)